sos/report/plugins/canonical_livepatch_onprem.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class CanonicaLivepatchOnprem(Plugin, UbuntuPlugin):
+
+    short_desc = 'Canonical Livepatch Onprem Service'
+
+    plugin_name = 'canonical_livepatch_onprem'
+    profiles = ('services',)
+    files = ('/usr/local/bin/livepatchd',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/livepatchd.yaml",
```

Comment:
lgtm, just a question: is there an api token or a user/pass we should obfuscate in this file?

Reply:
There are both, actually - user credentials and a token that authenticates against upstream livepatch.. User password is already encoded, but perhaps not a bad idea to obfuscate. Is that something we'd do in the plugin directly, or elsewhere in sos?

Reply:
You can do it directly in the plugin itself. Here's an example of obfuscation in a yaml file:

https://github.com/sosreport/sos/blob/main/sos/report/plugins/juju.py#L65

Reply:
Awesome, thanks for the link! I have some obfuscation code in now that looks like it's having the intended effect :slightly_smiling_face: